omegatrade-app/frontend/src/environments/environment.ts

```diff
@@ -1,5 +1,5 @@
 export const environment = {
   production: false,
   name: "dev",
-  baseUrl:"http://localhost:3000/api/v1/"
+  baseUrl:"https://omega-trade-backend-manage-simulations-i3nfvlaq3a-uw.a.run.app/api/v1/"
```

Comment:
Is this URL static? Or is it something that should be placed in a configuration file somewhere?

Reply:
yes, This URL is static in frontend environment file.